app/Http/Controllers/Api/MobileValidationController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\Crypt;
+use Libraries\Otp;
+
+class MobileValidationController extends Controller {
+	private const IDENTITY_VALIDITY = 900;

```

Comment:
Please change to CODE_EXPIRATION = 900 // Add the human readable time as comment